### PR TITLE
PIM-3305: add cookbook for localizers

### PR DESCRIPTION
**Pull request by skeleton**

### CODE REVIEW

**Comment by sumbobyboys** [comment]
GTM

File: cookbook/localization/index.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+How to Use the Localizers
+=========================
+
+Localizers are used to convert a localized attribute to the standard format.
+Eq: a decimal with a comma separator have to be convert with a dot separator before insert in database.
```
Comment (phaseinducer) [inline cookbook/localization/index.rst]: "before being inserted" ?

File: cookbook/localization/index.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+How to Register a New Localizer
+-------------------------------
+
+The Akeneo PIM comes with a number of localizers.
+It also comes with a flexible way to define your own localizer.
```
Comment (phaseinducer) [inline cookbook/localization/index.rst]: "The Akeneo PIM comes with several native localizers, but provides also a flexible way to define your own localizers." ?

File: cookbook/localization/index.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+.. code-block:: yaml
+
+    pim_localization.localizer.my_own_localizer:
+        class: %my_localizer_class%
+        arguments:
+            - ['pim_catalog_number']
+        tags:
+            - { name: pim_localization.localizer }
+
+The only argument of the service takes a list of attribute type you want to convert.
```
Comment (phaseinducer) [inline cookbook/localization/index.rst]: "**is** the list" ?
"attribute type**s**"

**Comment by phaseinducer** [comment]
Did you mix up "eq." and "e.g." ?

File: reference/import_export/product-import.rst [hunk 4]
```diff
@@ -288,6 +291,22 @@ The class ``Pim\Component\Connector\ArrayConverter\Flat\ProductStandardConverter
 
     The versionning will be reworked in a future version to use it too.
 
+Product Processor - AttributeLocalizedConverterInterface
+--------------------------------------------------------
+
+When you import a product with localized attributes (eq. prices with decimal separator as a comma),
```
Comment (phaseinducer) [inline reference/import_export/product-import.rst]: "e.g."
Rather "prices with comma as decimal separator" ?

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**